manifests/example/clone-block-pv.yaml

```diff
@@ -0,0 +1,24 @@
+kind: PersistentVolume
```

Comment:
I don't think we need multiple examples of how to create a block PV. The one from the importer should be sufficient.